pkg/bindings/containers/types.go

```diff
@@ -29,6 +29,7 @@ type LogOptions struct {
 type CommitOptions struct {
 	Author  *string
 	Changes []string
+	Config  []byte
```

Comment:
I think it would make more sense to accepts an io.Reader here instead.

Reply:
Had to make it a `*io.Reader` to make the generator happy.

Reply:
Yeah I think the generator could use some improvements, IMO a pointer to a interface makes things harder to use and more likely to cause nil derefs.
Anyway this shouldn't really block your work.